includes/classes/AdminCustomizations/EnvironmentIndicator.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Admin customizations
+ *
+ * @package  10up-experience
+ */
+
+namespace TenUpExperience\AdminCustomizations;
+
+use TenUpExperience\Singleton;
+
+/**
+ * Admin Customizations class
+ */
+class EnvironmentIndicator {
+
+	use Singleton;
+
+	/**
+	 * Setup module
+	 *
+	 * @since 1.7
+	 */
+	public function setup() {
+		// Add an admin bar item if in wp-admin.
+		add_action( 'admin_bar_menu', [ $this, 'add_toolbar_item' ], 7 );
+	}
+
+	/**
+	 * Add environment indicator to admin bar
+	 *
+	 * @param WP_Admin_Bar $admin_bar Admin bar instance
+	 */
+	public function add_toolbar_item( $admin_bar ) {
+		$environment = wp_get_environment_type();
```

Comment:
@psorensen If the `WP_ENVIRONMENT_TYPE` const and `WP_ENVIRONMENT_TYPE` env are not defined and we're on a `.local`, `.test` or `.localhost` domain, can we default this to `local`? 

Do you have thoughts, for/against that?

Reply:
My opinion is to align with what Core does here and default to production: https://github.com/WordPress/wordpress-develop/blob/6.6/src/wp-includes/load.php#L214

I think defaulting to production is the safest - imagine the const not being defined on a production environment - we wouldn't want it to display 'local'